### PR TITLE
feat(fontawesome): init

### PR DESCRIPTION
**Pull request by trinkey**
## 🎉 Theme for Font Awesome 🎉
An svg icon library that kinda sucks but not as much as their website's css

## 💬 Additional Comments 💬
- this websites uses variables. occasionally.
- sometimes inline styles redefine variables
- why are there so many inline styles
- the inline styles don't stop help

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/scripts/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - [x] `catppuccin.user.css` - all the CSS for the userstyle, based on the
        template.
  - [x] `preview.webp` - composite image of all four individual flavor screenshots (taken with the default accent color of mauve) stitched together, generated via [Catwalk](https://github.com/catppuccin/catwalk).

### CODE REVIEW

**Comment by trinkey** [comment]
404 page isn't styled ughhhhhhhhhhhhhh

**Comment by sgoudham** [comment]
Testing

File: scripts/userstyles.yml [hunk 1]
```diff
@@ -342,6 +342,13 @@ userstyles:
     readme:
       app-link: "https://elk.zone"
     current-maintainers: [*ryanccn]
+  fontawesome:
+    name: Font Awesome
+    categories: ["development", "photo_and_video"]
```
Comment (uncenter) [inline scripts/userstyles.yml:347]:
```suggestion
    categories: [development]
```

**Review by uncenter — CHANGES_REQUESTED** [review]
Stuff on https://fontawesome.com/search?ip=classic&s=regular&o=r

image

image

In modal on search page

image


image
(arrow buttons next/previous in modal)

https://docs.fontawesome.com/web/style/rotate:

image

(at bottom)
image

(sidebar active item background)

https://docs.fontawesome.com/web/style/mask:

image

https://docs.fontawesome.com/web/add-icons/upload-icons/icon-design:
image

Header and below text on https://docs.fontawesome.com/ (--sl-color-text-accent is unthemed)

**Comment by trinkey** [comment]
i see your feedback and will work on fixing these things when i have time but i'm like really busy right now, i'll hopefully get these done by tomorrow or saturday

**Comment by trinkey** [comment]
> i'll hopefully get these done by tomorrow or saturday

ill do it tomorrow i swear

**Comment by uncenter** [comment]
FYI the linter is failing.

**Review by uncenter — CHANGES_REQUESTED** [review]
CleanShot 2024-12-24 at 13 03 32

These codeblock tabs look inverted. The mantle color should be used for the code backgorund and inactive tabs, and the lighter color should be used for the active tab maybe?

**Review by isabelroses — CHANGES_REQUESTED** [review]
A few dark on dark buttons here.
image
image

We seem to have added our own backgrounds occasionally, which leads to wierd borders and strangely sized boxes.
image

When an icon is `:active`, it goes invisable
image

They also have code blocks, https://docs.fontawesome.com/apis/graphql/get-started, I do believe we have an expressive code port, well kind of https://github.com/expressive-code/expressive-code/blob/65e9b6b53936f53b147c1415a58d538e3aa3df35/docs/plugins/shiki-colorized-brackets/themes.js#L8 this is more of a in the future thing.

**Comment by trinkey** [comment]
i'm not going to continue this style because the website is a hot mess and i don't "enjoy" (as much as you *can* enjoy writing css) theming it. might pick it up again but unlikely

**Pull request closed without merging**